README.md

```
@@ -1,90 +1,98 @@
-🔴 Direct Alternative // FoobarZA's [FasterLoot](https://www.curseforge.com/wow/addons/fasterloot)
-
-🔴 Direct Alternative // LazyLoafs_Addons's [Lazy LockBoxes](https://www.curseforge.com/wow/addons/lazy-lockboxes)
-
-🔴 Direct Alternative // Dapal003's [Openable be gone](https://www.curseforge.com/wow/addons/auto-open-anything-updated)
-
-🔴 Direct Alternative // Yuyuli's [Speedy AutoLoot](https://www.curseforge.com/wow/addons/speedyautoloot)
+# Open Sesame
+
+Automatically open clams, lockboxes, and unlocked containers in your bags, no clicking required. Built-in Speedy Loot hides the loot window for near-instant auto looting. Less time clicking, more time playing.
+
+<img width="360" src="https://github.com/user-attachments/assets/48e86839-798f-4086-82d1-bcf9c23dc1eb" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)